docs/conf.py

```diff
@@ -98,7 +101,28 @@ def _get_project_meta():
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = "pydata_sphinx_theme"
+
+html_theme_options = {
+    "github_url": "https://github.com/OpenCOMPES/sed",
+    "primary_sidebar_end": ["indices.html"],
+    "navbar_center": ["version-switcher", "navbar-nav"],
+    "show_nav_level": 2,
+    "show_version_warning_banner": True,
+    # maybe better to use _static/switcher.json on github pages link instead of the following
+    "switcher": {
+        "json_url": "https://raw.githubusercontent.com/OpenCOMPES/docs/main/sed/switcher.json",
+        "version_match": version,
+    },
+    "content_footer_items": ["last-updated"],
+}
```

Comment:
I don't like the curent layout so much, which does not show the whole docs tree in the left sidebar (e.g., if you are in tutorials, you don't see the main menu entries). can this be changed to look similar to the old theme?

Reply:
It should be possible to change that, I believe. However, those entries are showing on the right side, which I find quite a cleaner layout. And I can directly navigate to other tutorials without issue.

Reply:
See right side
<img width="1459" alt="image" src="https://github.com/user-attachments/assets/0d8d17ad-7de9-4810-ab5d-1fd9e12dd002">

Reply:
What I mean is that these menues are all distributed over the page, i.e. main menu on the top, section menue on the left, page menu on the right. I find this very confusing, and liked the unique tree of the old docs much better

Reply:
Yes, but this seems to be how basically all modern projects in python are documented. See [numpy](https://numpy.org/doc/stable/index.html), [scipy](https://docs.scipy.org/doc/scipy/), [xarray](https://docs.xarray.dev/en/stable/), [dask](https://docs.dask.org/en/stable/). I think dask keeps the page menu on both left and right siden and some others might too. 
Since our library is based on these packages, I think it's more intuitive for user to follow such layout. We can of course change it but including complete heirarchy in the left menu clutters everything, I find.

Reply:
I can't follow. In the examples you provide, the left menu always stays the same, independent where you navigate. For your new layout, this is not the case. This is very confusing, and makes it difficult to get an overview of the docs. This is my point.

Reply:
We can probably keep the current layout, changing that appears to be not so straight-forward